dev/src/Command/AddComponentCommand.php

```diff
@@ -184,21 +213,160 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $composer->updateMainComposer();
         $composer->createComponentComposer($new->displayName, $new->githubRepo);
 
+        if ($input->getOption('run-owlbot')) {
+            $output->write("\n\nBuilding the library using Bazel\n");
+            $output->write($this->bazelBuildLibrary($new->shortName, $googleApisDir, $new->protoPath));
+            $output->write("\n\nCopying the Bazel output to the Google Cloud PHP directory\n");
+            $output->write($this->copyBazelOutToGoogleCloudPhp($new->componentName, $googleApisDir, $output));
+            $output->write("\n\nOwlbot post processing\n");
+            $output->write($this->postProcess());
+        }
+
         $output->writeln('');
         $output->writeln('');
         $output->writeln('Success!');
 
         return 0;
     }
 
-    private function loadProtoContent(string $proto): string
+    private function loadRawFileContent(string $file): string
     {
-        if (file_exists($proto)) {
-            return file_get_contents($proto);
+        if (file_exists($file)) {
+            return file_get_contents($file);
+        }
+        if (count(explode('googleapis/', $file)) == 1) {
+            $fileUrlPath = $file;
+        } else {
+            $fileUrlPath = explode('googleapis/', $file)[1];
```

Comment:
The `str_starts_with` function looks good to me. However, the use-case I see here are the following:
1. User provides a path, eg: `../googleapis/google/cloud/policysimulator/v1/simulator.proto`, In this case, we need to remove `../googleapis/` if the proto is not present in the path given by the user. You may ask why would the proto file is not present there. I was thinking what if the user tries with an old repository where this file is not present (lets say, user forgot to `pull` it from `main`) I may be overthinking here.
2. User provides actual file url. eg: `google/cloud/policysimulator/v1/simulator.proto`, in that case, we just need to use the given url, no need to remove anything from it.

If the first use-case don't need to consider, we don't even need to remove the `googleapis` as user either provide existing file or the url in the format we expects. Let me know what do you think.

Reply:
>  I may be overthinking here.

I think so. I think logic should be simple - we either accept a file path or a URL path relative to `googleapis/googleapis`. So if the proto file exists, then it's the former, and we treat it that way. Otherwise it's the latter, and we treat it that way. This is determined by a simple call to `file_exists($protoPath)`

Reply:
Changed